pkg/integrations/github/snippet-darwin.sh

```diff
@@ -0,0 +1,14 @@
+mkdir ~/actions-runner && cd ~/actions-runner
+curl -o actions-runner-osx.tar.gz -L {{ .RunnerURL }}
+tar xzf ./actions-runner-osx.tar.gz
+./config.sh --token {{ .Token }} --url {{ .RepoURL }} --name {{ .Name }} --unattended --replace --labels {{ .Labels }}
+./svc.sh install
+plistName=$(basename $(./svc.sh status | grep "plist$"))
+mkdir -p /Library/LaunchDaemons
+mv ~/Library/LaunchAgents/"${plistName}" /Library/LaunchDaemons/"${plistName}"
+
+plutil -replace UserName -string {{ .Username }} /Library/LaunchDaemons/"${plistName}"
```

Comment:
I guess this will allow to change the user which boots the daemon.

Reply:
yes, we replace the username, stdout and stderr after the tool creates the launchd plist